src/core/Form/StatusText/StatusText.test.tsx

```diff
@@ -10,11 +10,6 @@ describe('props', () => {
       const { container } = render(<StatusText>Test text</StatusText>);
       expect(container.firstChild).toHaveTextContent('Test text');
     });
-
-    it('is null when no children given', () => {
```

Comment:
Should there still be a test to check that it renders correctly even without children?

Reply:
Added a test for this.

Reply:
See other comments for better explanation, but I think we should always keep aria-live attribute in DOM.

Reply:
Commented elsewhere. Should now be fixed.